cdap-ui/app/cdap/components/FileBrowser/index.js

```diff
@@ -141,6 +142,7 @@ export default class FileBrowser extends Component {
   }
 
   goToPath = (path) => {
+    path = trimPrefixSlash(path);
```

Comment:
is this trimming the prefix slash or suffix? I think it should be suffix right?

Also, what would happen if the path is root (`/`), will it get trimmed as well?

Reply:
No we specifically look for two consecutive '//' and replace them with '/'. 
 Yes it is suffix will change the function name